.github/dependabot.yml

```diff
@@ -6,3 +6,18 @@ updates:
     directory: /
     schedule:
       interval: weekly
+
+  - package-ecosystem: docker
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
can you change all intervals to monthly?

Reply:
Sure!

Reply:
Resolved in 61e12db